### ISSUE TITLE
Looking for Mantainers

### ISSUE THREAD

**Issue by mgonto** [mgonto/restangular#1319]
Hey all :),

As you may have noticed, recently, I've not put the attention Restangular deserves. Lots of people have been using Restangular and despite it's at a stable point, there're still a few bugs / features that I'd like to see implemented. For example, making it compatible with newer Lodash version. 

However, recently I've changed roles and unfortunately I don't have the time to mantain Restangular as I used to, and I can't give Restangular the time it really deserves.

This is why @ConcurrentHashMap will now help solving some issues and merging PRs. However if somebody else wants to join him in helping work with PRs / new features, all the help is welcome to continue making this project awesome :).

Thanks all for the love <3

**Comment by pedromarce** [comment]
Hi @mgonto, first good luck in your new position, second, happy to help @ConcurrentHashMap as a maintainer in the project. If you could add me there.

Cheers,

**Comment by mgonto** [comment]
Thanks @pedromarce :).

**Comment by ConcurrentHashMap** [comment]
So, thank you for your trust @mgonto. I will do my best to take care of PRs and issues.

@pedromarce: Would you like to contact me via mail to organize us a little bit? :)

**Comment by pedromarce** [comment]
Sure @ConcurrentHashMap, could we use twitter (mine is @pedromarce)?

**Comment by daviesgeek** [comment]
@mgonto thanks for all your hard work

@ConcurrentHashMap thanks for coming on! I'd like to help as best I can. I'm unsure as to how much time I'll be able to dedicate to helping, but I'm quite familiar with Restangular and would love to help keep  making it awesome.

**Comment by ConcurrentHashMap** [comment]
:+1: Sounds great!

**Comment by daviesgeek** [comment]
@ConcurrentHashMap Is there a gitter/Slack channel or plans for one?

**Comment by pedromarce** [comment]
Good idea @daviesgeek, I have created restangular.slack.com, I will send invitations for yourself and @ConcurrentHashMap.

**Comment by hartz89** [comment]
I've been using Restangular for a little over a year, and more recently have actually been digging around and making some changes/enhancements in my own fork.  So, I've become familiar enough that I'd be comfortable contributing where I can.  If you guys still want/need the help, shoot me an invite to the Slack channel!

**Comment by pedromarce** [comment]
Hi there, sounds good to me, could you tell me where to send the invite? Cheers,

**Comment by hartz89** [comment]
ryan.hartzfeld@gmail.com

**Comment by simonvpe** [comment]
Does this mean there is no chance of seeing a restangular 2.0 anytime soon?

**Comment by bostrom** [comment]
@pedromarce @ConcurrentHashMap @daviesgeek I've been using restangular for over two years and I've also done some changes to it in my own fork. I couldn't say that I've got the expertise to continue on features, but I see that this github project would need some love. There's a lot of issues and PRs hanging around that would need to be resolved. If you need help with that, I'd be happy to contribute with a little of my time.

**Comment by daviesgeek** [comment]
@bostrom if you could help out with cleaning up some of the issues and PRs, that would be awesome. I just don't have the time right now :/

**Comment by bostrom** [comment]
@daviesgeek sure! I can spend some time on it now and then, perhaps this repo will get back in shape sooner or later. Are you actively discussing the state of things somewhere (slack/gitter)?

**Comment by daviesgeek** [comment]
@bostrom Yes we have a Slack channel. What's your email? I'll add you

**Comment by bostrom** [comment]
@daviesgeek fbostrom at gmail.

**Comment by daviesgeek** [comment]
@bostrom Thanks! Invite just sent out to ya!

**Comment by eugene-zheng0208** [comment]
Hi. everyone.
Is anyone used Restangular to post image or files to Laravel backend?
